lib/Models/GlobeOrMap.ts

```diff
@@ -158,9 +158,9 @@ export default abstract class GlobeOrMap {
     rectangle: Rectangle
   ): () => void;
 
-  _highlightFeature(feature: Feature | undefined) {
+  async _highlightFeature(feature: Feature | undefined) {
     if (isDefined(this._removeHighlightCallback)) {
-      this._removeHighlightCallback();
+      await this._removeHighlightCallback();
       this._removeHighlightCallback = undefined;
```

Comment:
`await` is required because if unset `_removeHighlightCallback` before the callback is complete, then we might break the functioning of the callback and features will fail to un-highlight.